macros/parserDifferenceQuotient.pl

```diff
@@ -15,28 +15,27 @@
 ################################################################################
 
 =head1 NAME
-
 parserDifferenceQuotient.pl - An answer checker for difference quotients.
-
 =head1 DESCRIPTION
-
```

Comment:
I don't think you need to do that.  You can just add commits to this one.  If you add them to this branch, they will become part of this pull request.  That will keep the comments together with the final pull request.